elastic/kibana
# [Uptime] Fix snapshot query to handle large numbers of monitors

## Summary

This PR seeks to allow the `Snapshot` component to display correct counts for relatively large numbers of HB monitors.

**Before:**
![image](https://user-images.githubusercontent.com/18429259/52446431-fc538280-2afb-11e9-9707-62ac6aeccfb1.png)


**After:** 
![image](https://user-images.githubusercontent.com/18429259/52445700-d3ca8900-2af9-11e9-84e1-4aafc59b326e.png)

As you can see, there is a dramatic improvement in the look and feel of the UI with this fix enabled.

## Testing this PR
- Start Heartbeat with several thousand monitors
- Load the UI
- Ensure that the counts in the snapshot correspond to what you expect
